frontend/src/api/requests.ts

```diff
@@ -114,7 +121,7 @@ export async function put(
   body: unknown,
   headers: Record<string, string> = {},
 ): Promise<Response> {
-  const response = await fetchRequest("GET", url, body, headers);
+  const response = await fetchRequest("PUT", API_BASE_URL + url, body, headers);
```

Comment:
@navidboloorian This was a little sloppy tbh, I should have looked more closely when reviewing